README.rst

```diff
@@ -258,6 +258,21 @@ Quick Example: How to train the **PETS Dataset**
    trainer = Trainer(max_epochs=3, gpus=1)
    trainer.fit(light_model, train_dl, valid_dl)
 
+Streamlit Demo
+--------------
+
```

Comment:
It would be good to add How to install streamlit, something like this:
--------------
If streamlit is not already install, run the following command, from the terminal to install it:

`pip install streamlit`

Reply:
Added.